tests/test_sampling_rates.py

```diff
@@ -138,6 +131,7 @@ def setup_sampling_decision(self):
     )
     @bug(context.library >= "python@1.11.0rc2.dev8", reason="Under investigation")
     @bug(library="golang", reason="Need investigation")
+    @bug(library="ruby", reason="Route /sample_rate_route not implemented")
```

Comment:
When something is missing on weblog, we usually caracterize the test as missing feature : 

```suggestion
    @missing_feature(library="ruby", reason="Route /sample_rate_route not implemented")
```